### PR TITLE
docs: added more categories and supported indexers

### PR DESCRIPTION
**Pull request by s0up4200**
New supported indexers:

- Aither
- Piratethenet
- PixelHD
- BitSexy
- Cathode-Ray Tube
- DocsPedia.world
- SkipTheCommericals

Added categories from Aither, PTN and PixelHD.

### CODE REVIEW

File: snippets/categories/ptn.mdx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<details><summary>Piratethenet</summary>
+
+- 2160P
+- 1080P
+- 720P
+- WebRip
```
Comment (martylukyy) [inline snippets/categories/ptn.mdx:6]: Is there only WebRip and then no other category such as WEB WEBDL or disc categories?

File: snippets/indexers.mdx [hunk 2]
```diff
@@ -2,23 +2,27 @@
   <summary>Click to view supported indexers</summary>
 
 - AcidLounge
+- Aither
 - AlphaRatio
 - AnimeBytes
 - BeyondHD
 - Bit-HDTV
-- BTFiles
```
Comment (martylukyy) [inline snippets/indexers.mdx:9]: Why did this get removed?

File: snippets/categories/ptn.mdx (hunk 1 above)
Comment (s0up4200) [reply snippets/categories/ptn.mdx:6]: Yep. This is what I've gathered from the announces so far.

File: snippets/indexers.mdx [hunk 3]
```diff
@@ -30,17 +34,20 @@
...
 - Orpheus
-- PornBay
+- PassThePopcorn
+- Piratethenet
+- PixelHD
 - PolishSource
 - PolishTracker
+- PornBay
 - PreToMe
-- PassThePopcorn
```
Comment (martylukyy) [inline snippets/indexers.mdx:37]: I can see why this got removed but technically we still supported them if they were to fix their IRC and RSS is still a thing so I would leave it in.

**Review by martylukyy — APPROVED** [review]
Send it! 🚀

**Pull request merged**